src/components/battles/animations/shared/Fainting.ts

```diff
@@ -39,18 +39,23 @@ export class Fainting<TGameStartr extends FullScreenPokemon> extends Component<T
             this.gameStarter.physics.getMidY(thing) + thing.height * thing.scale!,
             1,
             (): void => {
+                const playerName = this.gameStarter.itemsHolder.getItem("name");
+                const partyIsWipedText: (string | string[])[][][] = [];
                 this.gameStarter.physics.killNormal(thing);
                 this.gameStarter.physics.killNormal(blank);
                 this.gameStarter.menuGrapher.createMenu("GeneralText");
-                this.gameStarter.menuGrapher.addMenuDialog(
-                    "GeneralText",
-                    [
+                if (this.gameStarter.battles.isPartyWiped()) {
+                    partyIsWipedText.push([
                         [
-                            pokemon.nickname, " fainted!"
+                            pokemon.nickname, " fainted!\n",
+                            playerName, " is out of useable Pokemon!\n",
+                            playerName, " blacked out!"
                         ]
-                    ],
-                    onComplete
-                );
+                    ]);
+                }
+                partyIsWipedText.push([[pokemon.nickname, " fainted!"]]);
+
+                this.gameStarter.menuGrapher.addMenuDialog("GeneralText", partyIsWipedText[0], onComplete);
```

Comment:
Because if the party is wiped then the first index in the array will be the text for it and if it isn't then the first index in the array will be the just be the text "blank fainted"